web/js/general.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function () {
```

Comment:
In my opinion, we don't have to wait for a DOM ready event here:

 * The JavaScript is executed right before `</body>`. The DOM required for your mutations is all there.
 * The click event is bound to document.
 * Currently, it could create a FOUC on the my services page, as `my-services.js` is loaded (synchronously) by the browser, before the DOM ready event is fired.